### PR TITLE
Nerfs the Drake Armor and Explorer's Suit Plate Upgrades

### PR DESCRIPTION
**Pull request by IndieanaJones**
## About The Pull Request

This PR nerfs the ash drake's melee armor from 70 to 50, and it's laser armor from 50 to 30.  It also nerfs the explorer's suit goliath plate upgrades from giving 10 melee resistance to 5 melee resistance.

## Why It's Good For The Game

The armor values on the drake armor and the fully-upgraded explorer suit were pretty high for something not difficult to obtain.  The main appealing feature of the drake armor should be ash resistance, not insane armor resistances.  The issues mainly manifested when miners came back to the station with armor that made them incredibly durable, especially so considering that they have access to medipens and legion cores that work perfectly fine station side.  These healing items also work well enough on Lavaland, and this change shouldn't really impact the balance of Lavaland except for those who abuse armor values to facetank.

## Changelog
:cl:
balance: Lack of veterinarians in their diet have weakened the defensive properties of an ash drake's scales when used for armor.  Miners have also reported that goliath scales prove not as effective when attached to their explorer suit.
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
Well. I agree that armor values are generally pointless because 99.99 percent of all megafauna negate armor anyways.

But this seems like another nerf complaining about the disparity between mining balance and station balance.

Instead of making being a miner more difficult for anyone with high IQ ping so they never get hit. Maybe ALL of the balance should get some toning down.

Thus such disparities stop increasing.

Tldr; Chop down the tree not its branches.

**Comment by IndieanaJones** [comment]
> Well. I agree that armor values are generally pointless because 99.99 percent of all megafauna negate armor anyways.
> 
> But this seems like another nerf complaining about the disparity between mining balance and station balance.
> 
> Instead of making being a miner more difficult for anyone with high IQ ping so they never get hit. Maybe ALL of the balance should get some toning down.
> 
> Thus such disparities stop increasing.
> 
> Tldr; Chop down the tree not its branches.

This is more for just getting rid of number values which just end up being unnecessary for miners.  Because megafauna don't care and the lesser mobs don't do that much damage, there's no need for this much armor on either of these.

**Comment by Shadowflame909** [comment]
I think it's like that for a good reason though.

On-Station and lavaland balance are pretty much extremely different.

The most prime examples after ignoring the unfairness of megafauna who don't care about armor. It would be the watcher's Cryo-beam + slow-down, and the goliath's stun upon which he brutally tenderizes you.

It does add up, and that healing cactus has a ton of nutriment. Which fattens you pretty fast, adding a slowdown on top of all this.

Something needs to change for sure. But I don't think this is the right way to go about it.

Have any recent examples of on station balance with miner armor stuck out to you? Is someone power-gaming it?

**Comment by IndieanaJones** [comment]
> I think it's like that for a good reason though.
> 
> On-Station and lavaland balance are pretty much extremely different.
> 
> The most prime examples after ignoring the unfairness of megafauna who don't care about armor. It would be the watcher's Cryo-beam + slow-down, and the goliath's stun upon which he brutally tenderizes you.
> 
> It does add up, and that healing cactus has a ton of nutriment. Which fattens you pretty fast, adding a slowdown on top of all this.
> 
> Something needs to change for sure. But I don't think this is the right way to go about it.
> 
> Have any recent examples of on station balance with miner armor stuck out to you? Is someone power-gaming it?
> 
> Edit: It's harsh and cruel. So the rewards for surviving it are sweeter.
> 
> Maybe it's time to bring back VR: Lavaland edition to fix balance once and for all!1! Ores being mined via thought-waves.

Thing is, Ash Drake and goliaths are both easy kills.  Goliaths are a simple matter of kiting that even an extremely laggy person can do and ash drakes are the easiest megafauna to kill, giving the best reward.  There's a lot of things you can do to fix lavaland balance, but this is just cutting off some extra fat that honestly doesn't need to be there, that's all.

**Comment by Repukan** [comment]
If anything nerf the amount of plates it drops for only one set of armor.

I'd argue that the new drake is non-killable for 80% of the players and if you're good enough to see it's easy it'd be better to buff the drake and make it actually permakill you (Instead of the pussy copout they added with it keeping your head.)

**Comment by Repukan** [comment]
Explorer set armor is a relic from the past and can get fucked though

**Comment by Whoneedspacee** [comment]
i think it’s kind of unnecessary but considering the drake drops a couple sets its probably fair that its not better than unique armor sets

**Comment by Whoneedspacee** [comment]
also yeah the explorer suit being upgradeable is kind of stupid especially since there’s no visual indication you’re doing signifigantly less damage

**Comment by Learner10** [comment]
>  and the lesser mobs don't do that much damage, there's no need for this much armor on either of these. 

Goliaths will maul you horribly if they get the opportunity, Legion are most likely going to get a strike or two in, Watchers are annoying, but at least relatively passive. Everything else is just death incarnate, and this includes the new Lavaland elites. 

> but it just tones them down for station shennanigans, where these damage types matter.

What is the implication here? Does my life not matter when I'm the one risking death and permadeath just so the roboticist can make his DNA locked mech suits, and the engineering crew can have all of the stations metal for nothing I'll ever see? At the end of the day, one only uncommonly encounters a miner on-station, and when they do get seen, it's no guarantee that they're off to go smash someone's face in anyways. Yes, Miners are dangerous, but so is where they go to work. They just happen to come back a little more dangerous as well...

If anything, I have to agree that this sounds like a complaint about the difference between miners and non-miners. If true, then the issue is a matter of scaling. Bring the threat of anything that doesn't ignore armor on Lavaland down to match, and then it would be suitably justifiable to give up a few points of armor for the PVP encounters. 

Hell, the grand solution would be to implement a system to differentiate armor resistance against attacks from mobs, humans, and non-humans respectively. With such a thing, balancing Miners could be way less detrimental for their entire work, just for starters anyhow..

**Comment by Ratvar** [comment]
Or you could cut all lavaland mob attack and health in two, and then cut all miner armor and damage in two. And remove their fancy 30% stamina armor, radiation/viral armors and all other fancy things that are entirely unrelated to lavaland itself and fully related to sneaky coders.

**Comment by HomocidlePancakes** [comment]
>The armor values on the drake armor and the fully-upgraded explorer suit were pretty high for something not difficult to obtain.

Why wouldn't they be high and easy to obtain? It's kinda necessary with the hazards of the jobs. Why not gimp the rad protections on engineering gear? It's pretty high and not difficult to obtain.

>The main appealing feature of the drake armor should be ash resistance, not insane armor resistances.

By whos decree? Storm is easy to avoid and is more of an annoyance really. I'd take extra armor over the storm protections 90% of the time.

>The issues mainly manifested when miners came back to the station with armor that made them incredibly durable, especially so considering that they have access to medipens and legion cores that work perfectly fine station side. 

A job oriented at exploration and combat is hard to kill? Whaaaaaaaaaaat? Blasphemous! But seriously, why wouldn't they be super durable? It kinda comes with the job, doesn't it? Or an unprepared grey should have a fair chance in a fight against everyone in your opinion?

>this change shouldn't really impact the balance of Lavaland except for those who abuse armor values to facetank.

The only thing you can facetank is the hirophant, which deals burn damage, and the blood-drunk, which is low-risk low-reward (fix these btw). I guess by facetanking you mean facetanking regular fauna? They are barely a danger anyway.

>This is more for just getting rid of number values which just end up being unnecessary for miners
>Because megafauna don't care

Being able to endure one more hit while fighting megafauna is a big deal.

>and the lesser mobs don't do that much damage
>Goliaths are a simple matter of kiting that even an extremely laggy person can do

True. They are *lesser* mobs for a reason.

>ash drakes are the easiest megafauna to kill, giving the best reward for effort put in

A good chunk of people still struggle with it. It's not nearly as easy as you claim it to be.

**Comment by 81Denton** [comment]
>ash drakes are the easiest megafauna to kill

Not anymore since the buffs. I'd argue that Hierophant and Colossus are the easiest to smash.

**Comment by necromanceranne** [comment]
Hiero has always been easy, I don't think there is nuch disparity between reward and difficult with hiero.

However, worth reminding people that apparently in the early days of mining people would grab the riot suit in order to mine more effectively. The riot suit then was nerfed by adding a slowdown, and to compensate I believe that was when explorer suit upgrades were added. This is all second hand so I might have this horribly wrong so do forgive me. Anyway recently that slowdown on riot suits were reverted. And miner explorer suits are being nerfed back down below riot suit. 

The cycle of nerfs continues.

**Comment by HomocidlePancakes** [comment]
The slowdown you are talking about was added during the disarm rework. Explorer/hardsuit upgrades were in since the times of the asteroid.

**Comment by IndieanaJones** [comment]
I guess a different method should be considered.
